src/contracts/core/AllocationManager.sol

```diff
@@ -111,13 +111,14 @@ contract AllocationManager is
 
             // 5. Update state
             _updateAllocationInfo(params.operator, operatorSet.key(), strategy, info, allocation);
-            _updateMaxMagnitude(params.operator, strategy, info.maxMagnitude);
+            uint64 prevMaxMagnitude = _updateMaxMagnitude(params.operator, strategy, info.maxMagnitude);
```

Comment:
(or, its prior value, anyway)

Reply:
feels much better to assign `prevMaxMagnitude` higher in the method near `slashedMagnitude` and `sharesWadSlashed` rather than giving this method a new return value.

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/881/commits/173c411ca2bc66853c4f417d09557a35dc5a4935